docs/user/migrating-to-v1.0.0-alpha.5.md

```diff
@@ -39,7 +39,7 @@ foo 	1       	Mon Aug  6 12:10:07 2018	DEPLOYED	aerospike-0.1.7	default
 
 Now that we have backed up the releases we should delete existing Kubeapps resources. To do so execute:
 
-```
+```bash
 kubeapps down
```

Comment:
I'm assuming this is no longer relevant? (I've not seen a kubeapps binary since I've been on the project, but heard there was one originally). When do we delete obsolete docs... it may be that this is referred to in some release notes (or perhaps we referred to the tagged version which would be better). Do you think we can just delete this doc?

Reply:
Yes, this doc is old enough to be deleted

Reply:
As I see there are two options:
- A) We delete obsolete documentation (such as this)
- B) We create a folder and put the obsolete docs there